### ISSUE TITLE
Add text-formatting

### ISSUE THREAD

**Issue by flymia** [flymia/SpotCurrent#1]
Add text-formatting option (delimiter) for use with scrolling text in OBS. 

For example:

DJ Flux - Gold Dust DJ Flux - Gold Dust

But with a delmiter:
DJ Flux - Gold Dust || DJ Flux - Gold dust

**Comment by slfmessi** [comment]
Actually, scrolling text is available on OBS Studio by right-clicking on text and selecting "filters" and then add the "scroll" filter.﻿

**Comment by flymia** [comment]
Yeah I know. But what I meant is to add a || between it. So the currentsong.txt would look like this:

DJ Flux - Gold Dust ||

When it scrolls, it displays like this: DJ Flux - Gold Dust || DJ Flux - Gold dust

Do you know what I mean?

**Comment by slfmessi** [comment]
Like a separator which show where the song name ends?

**Comment by flymia** [comment]
Yes, but the user should be able to change it. It can be easily done by using a variable.
